Framework/Kernel/inc/MantidKernel/Statistics.h

```diff
@@ -43,6 +43,9 @@ struct Statistics {
   double median;
   /// standard_deviation of the values
   double standard_deviation;
+
+  /// Default value for everything is nan
+  Statistics();
```

Comment:
Now that there is an out-of-line constructor we need to export the symbols from DLL by defining the struct as
```c++
struct MANTID_KERNEL_DLL Statistics
```
That should sort the current build issues. We never enabled the symbol hiding on clang as it didn't support it at the time so that has passed.